version.php

```diff
@@ -1,4 +1,18 @@
 <?php
+/**
+ * Software version identification.
+ *
+ * @link http://open-emr.org/wiki
+ * @license https://github.com/openemr/openemr/blob/master/LICENSE GNU General Public License 3
+ * @author Robert Down <robertdown@live.com>
+ */
+
+require_once __DIR__ . "/vendor/autoload.php";
```

Comment:
This is likely going to likely break things in upgrade and setup scripts. Recommend doing a grep wherever version.php is used and testing those mechanisms extensively.